libc/test/src/stdio/sprintf_test.cpp

```diff
@@ -3493,8 +3493,8 @@ TEST_F(LlvmLibcSPrintfTest, FixedConv) {
   ASSERT_STREQ_LEN(written, buff, "       0.100 256.000     ");
 
   written =
-      LIBC_NAMESPACE::sprintf(buff, "%+-#12.3lk % 012.3k", 0x000000001013a92a,
-                              0x02740000); // 0.126, 1256.0
+      LIBC_NAMESPACE::sprintf(buff, "%+-#12.3lk % 012.3k", 0x000000001013a92all,
+                              0x02740000ll); // 0.126, 1256.0
```

Comment:
does this one need the `ll`? It looks like this is for the `% 012.3k` which doesn't have a length modifier.

Reply:
it doesn't, removed.